trunk/SUAVE/Methods/Aerodynamics/Airfoil_Panel_Method/infl_coeff.py

```diff
@@ -0,0 +1,79 @@
+## @ingroup Methods-Aerodynamics-Airfoil_Panel_Method
+# infl_coeff.py  
+
+# Created:  Mar 2021, M. Clarke
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+import SUAVE
+from SUAVE.Core import Units
+import numpy as np
+
+# ----------------------------------------------------------------------
+# infl_coeff.py
+# ----------------------------------------------------------------------  
+## @ingroup Methods-Aerodynamics-Airfoil_Panel_Method
+def infl_coeff(x,y,xbar,ybar,st,ct,npanel,nalpha,nRe,batch_analyis):
+    """Compute the matrix of aerodynamic influence  coefficients for later use
+
+    Assumptions:
+    None
+
+    Source:
+    None 
+ 
+    Inputs
+    x       -  Vector of x coordinates of the surface nodes     
+    y       -  Vector of y coordinates of the surface nodes      
+    xbar    -  x-coordinate of the midpoint of each panel     
+    ybar    -  y-coordinate of the midpoint of each panel     
+    st      -  np.sin(theta) for each panel                                
+    ct      -  np.cos(theta) for each panel                             
+    npanel  -  Number of panels on the airfoil                      
+                                                                                            
+    Outputs                                        
+    ainfl   -  Aero influence coefficient matrix    
+
+    Properties Used:
+    N/A
+    """                          
+   
+    ainfl                = np.zeros((nalpha,nRe,npanel+1,npanel+1))    
+    pi2inv               = 1 / (2*np.pi) 
+    
+    # convert 1d matrices to 2d 
+    x_2d                 = np.repeat(np.swapaxes(np.swapaxes(x,0, 2),0,1)[:,:,np.newaxis,:],npanel, axis = 2)
```

Comment:
its already in 2d, I actually do a lot of manipulation because if we have different airfoils/different flow conditions, the surface velocities will be different (down the road). These lines allow me to do a 4 d computations in a few lines below as opposed to having a lot for loops - hence I said during the last meeting that I got vectorization done....but its nasty